src/ai_client.c

```diff
@@ -3,6 +3,8 @@
  *
  * Copyright (C) 2009-2011 Keldon Jones
  *
+ * Source file modified by J.-R. Reinhard, October 2016.
```

Comment:
Yes, this change does not impact any communication with old servers (without XI).